app/models/review.rb

```diff
@@ -8,4 +8,8 @@ class Review < ApplicationRecord
   def self.average_review_rating
     average(:rating)
   end
+
+  def self.top_three_reviews
+    order(:rating).reverse[0..2].pluck(:title, :rating)
```

Comment:
Good work on discovering the pluck method! This really helped us with the top three and bottom three review methods